ovirt/util.go

```diff
@@ -10,6 +10,17 @@ import (
 	ovirtclient "github.com/ovirt/go-ovirt-client"
 )
 
+func schemaMerge(schema1, schema2 map[string]*schema.Schema) map[string]*schema.Schema {
+	result := make(map[string]*schema.Schema, len(schema1)+len(schema2))
+	for k, v := range schema1 {
+		result[k] = v
+	}
+	for k, v := range schema2 {
```

Comment:
If schema1 and schema2 have the same key, schema2 would "win" and overwrite the value from schema1. Should we add a small log entry or is this expected behavior and, therefore, does not require this information?

Reply:
We could, but I don't think it's necessary.